flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/history/HistoryServerTest.java

```diff
@@ -105,10 +109,13 @@ public void setUp() throws Exception {
 	}
 
 	@After
-	public void tearDown() {
+	public void tearDown() throws IOException {
 		if (cluster != null) {
 			cluster.after();
 		}
+
+		FileUtils.deleteDirectory(jmDirectory);
```

Comment:
redundant, `TemporaryFolder` takes care of it

Reply:
Removed but I had to replace `@ClassRule` by `@Rule` and change `TMP` from constant to property `tmpFolder` because `@ClassRule` is applied after all tests.

Reply:
That should be fine.